front/app/containers/Authentication/useSteps/stepConfig/missingDataFlow.ts

```diff
@@ -135,6 +155,45 @@ export const missingDataFlow = (
             onSuccess: async () => {
               const { requirements } = await getRequirements();
 
+              if (showOnboarding(requirements.onboarding)) {
+                setCurrentStep('missing-data:onboarding');
+                return;
+              }
+
+              if (requirements.special.group_membership === 'require') {
```

Comment:
What was the reason for adding this everywhere, was there a bug related to verification? I thought that this should already be handled inside of the verification step, and if you are already signed in the action descriptors should be making sure that you can never do an action (e.g. post an idea) for which you don't have the correct group membership, and so the authentication flow should never fire. Or at least that was the philosophy behind this when we designed it, but I guess we missed something?